.github/workflows/build.yml

```diff
@@ -284,8 +284,8 @@ jobs:
         id: build
         uses: nick-fields/retry@v3
         with:
-          timeout_minutes: 360
-          max_attempts: 3
+          timeout_minutes: 150
+          max_attempts: 2
```

Comment:
The warning on line 703 does not have relationship with this PR, but it is also shown here. Probably we can also address it here.